contracts/libs/gov-pool/GovPoolView.sol

```diff
@@ -133,4 +136,24 @@ library GovPoolView {
         unlockedIds = unlockedProposals.transform().crop(unlockedLength);
         lockedIds = lockedProposals.transform().crop(lockedLength);
     }
+
+    function getProposals(
+        mapping(uint256 => IGovPool.Proposal) storage proposals,
+        uint256 offset,
+        uint256 limit
+    ) internal view returns (IGovPool.ProposalView[] memory proposalViews) {
+        IGovPool govPool = IGovPool(address(this));
+
+        uint256 to = (offset + limit).min(govPool.latestProposalId()).max(offset);
+
+        (, , address validators, ) = govPool.getHelperContracts();
+
+        proposalViews = new IGovPool.ProposalView[](to - offset);
+        for (uint256 i = offset; i < to; ++i) {
+            proposalViews[i - offset] = IGovPool.ProposalView({
+                proposal: proposals[i],
```

Comment:
Probably we should add `1` to `i` in order to fetch the correct proposal. They start from the index `1`, but `i` starts from `0`